style.css

```diff
@@ -968,376 +624,478 @@ body {
-    font-style: normal;
-    font-weight: 700;
-    font-size: 23px;
-    line-height: 30px;
-    align-items: center;
-    color: #fff5e1;
-    flex: none;
-    flex-grow: 0;
-    margin-bottom: 0;
+  .works-card-desktop {
+    width: 100%;
+    padding-bottom: 60px;
+    display: flex;
+    margin-top: 8rem;
+    margin-bottom: 5rem;
   }
 
-  .headline p {
-    width: 325px;
-    height: 125px;
-    font-family: 'Poppins', sans-serif;
-    font-style: normal;
-    font-weight: 400;
-    font-size: 16px;
-    line-height: 25px;
-    color: #fff;
-    flex: none;
-    order: 2;
-    flex-grow: 0;
-    margin-top: 0;
-    margin-bottom: 0;
+  .extras {
+    width: 90%;
   }
 
-  .image-icons {
-    display: flex;
-    color: black;
-    flex-direction: row;
-    align-items: center;
-    padding: 0;
-    width: 200px;
-    height: 40px;
-    flex: none;
-    order: 3;
-    gap: 18px;
+  .side-text {
+    padding-left: 3rem;
+    width: 50%;
+    height: 50%;
   }
 
-  /* works section */
+  .grid-container {
+    display: grid;
+    grid-template-columns: repeat(3, auto);
+    grid-template-rows: repeat(2, auto);
+    column-gap: 1rem;
+    width: 95%;
+    row-gap: 2rem;
+    margin-top: -8rem;
+  }
 
-  button {
-    font-family: 'Crete Round', sans-serif;
-    font-weight: 700;
-    font-size: 17px;
-    line-height: 24px;
-    letter-spacing: 0.03em;
-    text-align: center;
-    color: #fff;
-    border: none;
+  .grid-container:hover {
     cursor: pointer;
   }
 
-  .card-body button {
-    background: #ff6b00;
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    text-align: center;
-    letter-spacing: 0.03em;
-    color: #fff;
-    padding: 12px;
-    gap: 10px;
-    width: 135px;
-    height: 48px;
-    flex: none;
-    order: 3;
-    flex-grow: 0;
-    font-family: 'Crete Round', sans-serif;
+  .grid-item {
+    position: relative;
+  }
+
+  .project-cards-mobile {
+    display: none;
+  }
+
+  .project-cards-desktop {
+    display: grid;
+    width: 100%;
+  }
+
+  .project-cards {
+    width: 755px;
+  }
+
+  .btn-type-b {
+    font-family: Inter, sans-serif;
     font-style: normal;
     font-weight: 700;
     font-size: 17px;
     line-height: 24px;
+    color: #fff;
+    align-items: center;
+    padding: 12px;
+    background-color: #f55800;
+    margin-top: 12px;
     border: none;
+    outline: none;
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    justify-content: center;
+    transition: background-color 2s;
   }
 
-  .card button {
-    background: #ff6b00;
-    display: flex;
+  .btn-type-b:hover {
+    background: #fff5e1;
+    color: #f55800;
+    border: 2px solid #f55800;
+    cursor: pointer;
+  }
+
+  .btn-type-b:active {
+    background-color: #e05304;
+  }
+
+  #details-popup {
+    position: fixed;
+    width: 90vw;
+    height: 80vh;
+    top: 15%;
+    left: 5%;
+    background-color: white;
+  }
+
+  .popup-header {
+    padding-bottom: 0;
+  }
+
+  .popup-body {
```

Comment:
- [ ] I would like to highlight some things in the `desktop view` of the popup cards too, kindly check the highlight below

- Kindly remove the `<br>` you gave the header text so that it can be on a straight line and not broken as seen in the Figma design
- Secondly the space between the image and the text is much kindly reduce the padding-left to 1rem
- Also the buttons as seen in the Figma design align with the image in the bottom, i would suggest you make the margin top in the class `.demoButtons` to be `43%`, 
- On the class `.btn-type-a` kindly give it a `text-decoration: none;` to remove the underline in the button
- On the class `.popup-text` kindly give it a `width: 90%`
- Kindly give a blur background as we have it in the Figma design

| Your Design  | Figma Design |
| ------------- | ------------- |
| ![san_LI](https://user-images.githubusercontent.com/78634841/233338845-ca966bf1-988e-435d-9ef9-d72d5074a397.jpg) | ![real](https://user-images.githubusercontent.com/78634841/233339134-13bce593-8db3-4c0a-9afb-129eb6645bd3.png)  |